crates/types/src/image.rs

```diff
@@ -196,8 +213,215 @@ impl Index<Point2<usize>> for Image {
     }
 }
 
+fn rgb_image_from_buffer_422(buffer: &[YCbCr422], width_422: u32, height: u32) -> RgbImage {
```

Comment:
I named `rgb_image` after the `RgbImage` type from the `image` crate. This should be kept.

The `422` of `buffer_422` is a specialization of the buffer and is therefore a suffix. We also have `width_422` because "it is mainly a width ... and btw it is in 422 format".

Let's keep it. Resolving.